### ISSUE TITLE
Lot of timing error in logs

### ISSUE THREAD

**Issue by darrepac** [mstojek/nlbw2collectd#3]
While the date of my openWRT system is correct, I have a lot of following errors in my logs...
```
Wed Dec 13 17:13:15 2023 daemon.err collectd[16061]: rrdtool plugin: rrd_update_r failed: /tmp/rrd/OpenWrt/iptables-mangle-iptmon_tx/ipt_bytes-tx_*.rrd: illegal attempt to update using time 1702483995 when last update time is 1702483995 (minimum one second step)
Wed Dec 13 17:13:15 2023 daemon.err collectd[16061]: rrdtool plugin: rrd_update_r failed: /tmp/rrd/OpenWrt/iptables-mangle-iptmon_rx/ipt_bytes-rx_*.rrd: illegal attempt to update using time 1702483995 when last update time is 1702483995 (minimum one second step)
Wed Dec 13 17:13:15 2023 daemon.err collectd[16061]: rrdtool plugin: rrd_update_r failed: /tmp/rrd/OpenWrt/iptables-mangle-iptmon_tx/ipt_packets-tx_*.rrd: illegal attempt to update using time 1702483995 when last update time is 1702483995 (minimum one second step)
Wed Dec 13 17:13:15 2023 daemon.err collectd[16061]: rrdtool plugin: rrd_update_r failed: /tmp/rrd/OpenWrt/iptables-mangle-iptmon_rx/ipt_packets-rx_*.rrd: illegal attempt to update using time 1702483995 when last update time is 1702483995 (minimum one second step)
Wed Dec 13 17:13:15 2023 daemon.err collectd[16061]: rrdtool plugin: rrd_update_r failed: /tmp/rrd/OpenWrt/iptables-mangle-iptmon_rx/ipt_bytes-rx_*.rrd: illegal attempt to update using time 1702483995 when last update time is 1702483995 (minimum one second step)
Wed Dec 13 17:13:15 2023 daemon.err collectd[16061]: rrdtool plugin: rrd_update_r failed: /tmp/rrd/OpenWrt/iptables-mangle-iptmon_tx/ipt_packets-tx_*.rrd: illegal attempt to update using time 1702483995 when last update time is 1702483995 (minimum one second step)
Wed Dec 13 17:13:15 2023 daemon.err collectd[16061]: rrdtool plugin: rrd_update_r failed: /tmp/rrd/OpenWrt/iptables-mangle-iptmon_rx/ipt_packets-rx_*.rrd: illegal attempt to update using time 1702483995 when last update time is 1702483995 (minimum one second step)
Wed Dec 13 17:13:15 2023 daemon.err collectd[16061]: rrdtool plugin: rrd_update_r failed: /tmp/rrd/OpenWrt/iptables-mangle-iptmon_tx/ipt_bytes-tx_*.rrd: illegal attempt to update using time 1702483995 when last update time is 1702483995 (minimum one second step)
```

**Comment by mstojek** [comment]
This star in the log ("tx_*.rrd") was put by yourself to remove sensitive data or you just copied original log? See fragment below:
[...]tmp/rrd/OpenWrt/iptables-mangle-iptmon_tx/ipt_bytes-tx_*.rrd[...]

If you copied logs without modifying them I need to dig I to this. In your network hosts have just ip addresses or both, ip and host names? Who is assigning ip and host names, Openwrt device or you have separate Dns/Dhcp server that does this?

**Comment by mstojek** [comment]
Probably it is related to https://github.com/oofnikj/iptmon/pull/17.
I will try to add fix for this tomorrow and let you know.

**Comment by darrepac** [comment]
> This star in the log ("tx__.rrd") was put by yourself to remove sensitive data or you just copied original log? See fragment below: [...]tmp/rrd/OpenWrt/iptables-mangle-iptmon_tx/ipt_bytes-tx__.rrd[...]
> 
> If you copied logs without modifying them I need to dig I to this. In your network hosts have just ip addresses or both, ip and host names? Who is assigning ip and host names, Openwrt device or you have separate Dns/Dhcp server that does this?

I just copy/past the log so you can have a look ;)
IP and host name are assigned by the device on which collectd is running (DHCP + static ip assigned)

**Comment by mstojek** [comment]
Check latest version. Let us see if the fix will also help with https://github.com/mstojek/nlbw2collectd/issues/4

**Comment by darrepac** [comment]
> Check latest version. Let us see if the fix will also help with #4

no more problems in log but not better for #4
